### PR TITLE
[docs] Edit the Pickers Getting started doc

### PR DESCRIPTION
**Pull request by mapache-salvaje**
- fix typos and grammar mistakes
- copyedits for clarity and brevity throughout
- apply style guide
- move some content here from Base concepts page - will follow up with another PR for that doc

### CODE REVIEW

**Review by flaviendelangle — APPROVED** [review]

File: docs/data/date-pickers/getting-started/getting-started.md [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
+:::success
+Not sure which date library to choose?
+If you're starting from scratch with no other date libraries in your app, we recommend Day.js because of its small bundle size.
 :::
 
-The Date and Time Pickers package has a peer dependency on `@mui/material`.
-If you are not already using it in your project, you can install it with:
+### Peer dependencies
+
+#### Material UI
```
[the remark this answers is not in the document]
Comment (michelengelen) [reply docs/data/date-pickers/getting-started/getting-started.md:32]: the good'ol NBSP :P

File: docs/data/date-pickers/getting-started/getting-started.md [hunk 2]
```diff
@@ -1,34 +1,38 @@
...
 githubLabel: 'component: pickers'
 materialDesign: https://m2.material.io/components/date-pickers
 waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/
 ---
 
-# Date and Time Pickers - Getting Started
+# Date and Time Pickers - Getting started
 
-<p class="description">Get started with the Date and Time Pickers. Install the package, configure your application and start using the components.</p>
+<p class="description">Install the Date and Time Pickers and set up your date library to start building.</p>
```
Comment (michelengelen) [inline docs/data/date-pickers/getting-started/getting-started.md:12]: shouldn't we include "package" here as well? Sounds wrong.
```suggestion
<p class="description">Install the Date and Time Pickers package and set up your date library to start building.</p>
```

File: docs/data/date-pickers/getting-started/getting-started.md [hunk 3]
```diff
@@ -1,34 +1,38 @@
...
+Choose your packages and manager through the toggles below, then run the commands as provided to install:
 
 <!-- #default-branch-switch -->
 
 {{"component": "modules/components/PickersInstallationInstructions.js"}}
 
-:::info
-If you need more information about the date library supported by the Date and Time Pickers,
-take a look at the [dedicated section](/x/react-date-pickers/base-concepts/#date-library)
+:::success
```
[the remark this answers is not in the document]
Comment (michelengelen) [reply docs/data/date-pickers/getting-started/getting-started.md:25]: @LukasTy related to our topic from yesterday. With all the problems dayJs has regarding timezones and its relatively unmaintained state are we sure we want to recommend that?

File: docs/data/date-pickers/getting-started/getting-started.md [hunk 4]
```diff
@@ -57,70 +63,40 @@ Please note that [react](https://www.npmjs.com/package/react) and [react-dom](ht
...
 
-```bash npm
-npm install @mui/styled-engine-sc styled-components
-```
+To integrate your chosen date library with the Date and Time Pickers, you'll need to plug the corresponding adapter into a `LocalizationProvider` that wraps your Picker components.
 
-```bash pnpm
-pnpm add @mui/styled-engine-sc styled-components
-```
+Adapters are provided for each of the supported libraries, and all are exported by both `@mui/x-date-pickers` and `@mui/x-date-pickers-pro`—for example,`AdapterDayjs`, which is used throughout this documentation for [Day.js](https://day.js.org/) integration:
```
Comment (michelengelen) [inline docs/data/date-pickers/getting-started/getting-started.md:70]: same as above ☝🏼

Comment (LukasTy) [reply docs/data/date-pickers/getting-started/getting-started.md:70]: I think that it is a separate topic from this PR that we should re-access and change the approach to if necessary. 😉

Comment (michelengelen) [reply docs/data/date-pickers/getting-started/getting-started.md:70]: Absolutely ... just wanted to point that out that everyone is aware of it :D

**Pull request merged**